configs/diffusion_hippocampus.py

```diff
@@ -44,9 +43,10 @@ def get_context(device, variables, predict_hbt=False, **kwargs):
     cohorts['cbbrain'] = RequireAttributes({"protocol": "cbbrain"})
     cohorts['ab300'] = RequireAttributes({"protocol": "ab300"})
     cohorts['rescans'] = ForbidAttributes({"rescan_id": "None"})
+    cohorts['labeled'] = RequireAttributes(['y'])
```

Comment:
Validations filters are applied after the transformations so the RenameProperty deleted the output variable is use